docs/docs/configuration/index.md

```diff
@@ -20,8 +20,8 @@ cameras:
           roles:
             - detect
     detect:
-      width: 1280
-      height: 720
+      width: # <- optional, by default Frigate try to automatically detect resolution 
```

Comment:
This part can probably just be removed. 

also https://github.com/blakeblackshear/frigate/blob/dev/docs/docs/configuration/index.md?plain=1#L196-L199 needs to be updated

Reply:
Sure, but this specific part is saying that the user should use a minimal config. and as the other parts of the docs recommend, the width / height are not part of the minimal config. 

as I linked to, the "full config" for detect is further down in the file